packages/generators/add-generator.ts

```diff
@@ -93,7 +47,8 @@ export default class AddGenerator extends Generator {
 		config: {
 			configName?: string,
 			topScope?: string[],
-			item?: string;
+			item?: string,
+			merge?: IWebpackOptions,
 			webpackOptions?: IWebpackOptions,
```

Comment:
merge is in webpackOptions, feel free to make it a config prop but then the ast checking for merge needs to be removed. This is nice if we want to spit out a pure webpack config in webpackOptions

Reply:
@evenstensberg I also like to emit a pure config in final configuration built,
But with `merge` prop in `config` (like in current scenario of this PR), final configuration is still having merge prop in `module.exports`. Is that has to be changed too?

Reply:
No merge in module exports in my point being. This is ok but ast needs to be checked for code involving this

Reply:
The config has `merge` as prop only and is not in `webpackOptions` as you said [here](https://github.com/webpack/webpack-cli/blob/master/packages/utils/modify-config-helper.ts#discussion_r268748106)!
and `runTransform()` also consider `merge` as prop in config prior to my changes too.
https://github.com/webpack/webpack-cli/blob/b96857f182481623db2c1fc799fdab59dab81107/packages/utils/modify-config-helper.ts#L11-L19

So, IMO problem is solved as standalone AST parsing checking of merge is insignificant as it may be using functions outside the scope of object, thus parsing of final file makes more sense.